test/integration/integration_config.yml

```diff
@@ -1,3 +0,0 @@
----
```

Comment:
Why was this file removed?

Reply:
Ah, that's the reason why the tests failed. I was really intending to clear out any changes we made to the file. Oops.